python/ray/worker.py

```diff
@@ -973,6 +976,9 @@ def _handle_process_task_failure(self, function_id, return_object_ids,
                 "function_id": function_id.id(),
                 "function_name": function_name
             })
+        # Mark the actor init as failed
+        if self.actor_id and function_name == "__init__":
```

Comment:
doesn't `self.actor_id` default to `NIL_ACTOR_ID` (which doesn't evaluate to None)?

Reply:
Fixed.